app/assets/stylesheets/mixins/_margins.scss

```diff
@@ -1,15 +1,15 @@
 @mixin responsive-bottom-margin {
   margin-bottom: govuk-spacing(3);
 
-  @include media(tablet) {
+  @include govuk-media-query($from: tablet) {
```

Comment:
Could these margins be replaced with the [`govuk-responsive-margin()`](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/helpers/_spacing.scss#L101)?

Reply:
I'm not sure. The existing mixins give a margin bottom/top of 15px on mobile and whatever 7 is above that (40px?). How would the govuk mixin replace that?

Reply:
Ah, true - if you want it exactly the same you can't.  Would a 25px margin on small screens be acceptable instead of a 15px margin? Then `@include govuk-responsive-padding(7)` could simplify things. Or `5` would give a 25px margin on large screens and 15px on small screens.

Reply:
I'm not familiar enough with the service manual to be comfortable making that change. I think it's definitely something to revisit later though.